dotcom-rendering/src/client/ga/ga.ts

```diff
@@ -105,7 +105,7 @@ export const sendPageView = async (): Promise<void> => {
 	ga(set, 'dimension9', GAData?.keywordIds);
 	ga(set, 'dimension10', GAData?.toneIds);
 	ga(set, 'dimension11', GAData?.seriesId);
-	ga(set, 'dimension16', (userCookie && 'true') || 'false');
+	ga(set, 'dimension16', (userCookie && 'true') ?? 'false');
```

Comment:
I think this changes the logic? Would it be more appropriate to coerce to boolean?

```suggestion
	ga(set, 'dimension16', !!userCookie ? 'true' : 'false';
```